packages/text-annotator/src/SelectionHandler.ts

```diff
@@ -113,10 +128,9 @@ export const SelectionHandler = (
       // select events don't have offsetX/offsetY - reuse last up/down)
       selection.userSelect(currentTarget.annotation, lastPointerDown);
     }
-  })
+  }, 10).bind(undefined);
```

Comment:
`.bind(undefined)` is required to prevent the `new Error("Debounced method called with different contexts.` exception. When we supply the denounced function directly to the `resize` event listener, it might be invoked with a different context. And the `debounce` library explicitly prohibits that - https://github.com/sindresorhus/debounce/issues/8

Reply:
That's the wrong approach... The `.bind()` method clears out all the custom properties and methods assigned to the `debounce` function: 
![image](https://github.com/user-attachments/assets/d8347435-5f0c-4570-b6cf-5357314f757b)
That will lead to the following exceptions: 
![image](https://github.com/user-attachments/assets/1d869f85-2baf-4dd9-aff6-65124e2286af)